docs/contributing/documentation-style-guide.md

```diff
@@ -95,38 +97,45 @@ the [Quick Start Guide](../getting-started/quick-start-guide.md).
 
 **Example**
 
-Less like this: we can use the following plugins.
+Less like this:
+>We can use the following plugins.
```

Comment:
the adding of quote blocks for examples was to support highlighting of the examples.